website/content/userguide/import_model.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Steps to Import Model"
+date: 2018-12-02T10:58:37+08:00
+weight: 107
+---
+
+In this page you will shown how to import an existing model.
+<!--more-->
+
+## Prerequisite
+
+If any user who needs to import any model(s), he/she need to be logged in to the Vorto repository.
```

Comment:
Keep prerequisites short and crisp, e.g.
- Successfully logged into the Vorto Repository